modules/mauth-signer/src/test/scala/com/mdsol/mauth/SignerProtocolSuiteSpec.scala

```diff
@@ -0,0 +1,82 @@
+package com.mdsol.mauth
+
+import java.security.Security
+import java.util.UUID
+
+import com.mdsol.mauth.test.utils.ProtocolTestSuiteHelper
+import com.mdsol.mauth.util.MAuthKeysHelper.getPrivateKeyFromString
+import com.mdsol.mauth.util.{EpochTimeProvider, MAuthSignatureHelper}
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+import scala.collection.JavaConverters._
```

Comment:
This is deprecated - can you pull in https://github.com/scala/scala-library-compat which will provide `import scala.collection.JavaConverters._` for both scala 2.12 and 2.13?